src/Bridge/Symfony/Bundle/Resources/config/api.xml

```diff
@@ -91,14 +90,31 @@
         <service id="api_platform.operation_path_resolver.router" class="ApiPlatform\Core\Bridge\Symfony\Routing\RouterOperationPathResolver" public="false">
             <argument type="service" id="api_platform.router" />
             <argument type="service" id="api_platform.operation_path_resolver.custom" />
+            <argument type="service" id="api_platform.subresource_operation_factory" />
         </service>
 
         <service id="api_platform.operation_path_resolver.custom" class="ApiPlatform\Core\PathResolver\CustomOperationPathResolver" public="false">
+            <!-- @TODO should be "api_platform.operation_path_resolver.generator" when the default is removed -->
```

Comment:
Can you remove the `@todo` annotation? It triggers annoying warnings in SensioLabs Insight.